src/dsp_tools/schemas/data.xsd

```diff
@@ -72,15 +72,15 @@
         <xs:attribute name="encoding" type="encoding_types" use="required"/>
         <xs:attribute name="resrefs" type="resrefs_type"/>
         <xs:attribute name="comment" type="xs:string"/>
-        <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
+        <xs:attribute name="permissions" type="xs:NCName" use="optional" default="prop-default"/>
```

Comment:
That's true. I made this modification with the status quo in my head. 
At some point of time in the future, this won't be necessary any more. But in the meantime, it could perhaps make your life easier when you create an XML.

Reply:
I don't think this will make a huge difference, but it can't hurt either, so I'm absolutely happy with having it in there